### PR TITLE
Add some FTS info

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Describe how to expose storage with GridFTP for FTS

---

**Related issue :**

### CODE REVIEW

File: content/en/users/cloud-compute/storage/_index.md [hunk 1]
```diff
@@ -86,3 +86,61 @@ Once you have a filesystem you can mount it at the desired path:
...
+- You need to specify the environment variables in the `docker-compose.yml` file
+  for obtaining the Let's Encrypt certificate. An extra variable
+  `GLOBUS_HOSTNAME` must be also set:
+
+  ```yaml
+  environment:
+    - TESTCERT
+    - EMAIL=youremail@domain.com
+    - DOMAIN=mygridftp.example.com
+    - GLOBUS_HOSTNAME=mygridftp.example.com
```
Comment (andrea-manzi) [inline content/en/users/cloud-compute/storage/_index.md:133]: is GLOBUS_HOSTNAME also needed? on the code i see that GLOBUS_HOSTNAME is set the value of DOMAIN https://github.com/cern-fts/ready2go/blob/master/gridftp-le/Dockerfile#L30

Comment (andrea-manzi) [reply content/en/users/cloud-compute/storage/_index.md:133]: actually i think there is a but there , as it should be `ENV GLOBUS_HOSTNAME=$DOMAIN`

Comment (enolfc) [reply content/en/users/cloud-compute/storage/_index.md:133]: it's needed, as the Dockerfile sets the value but that's container build time, on runtime if it was not properly set before, you will still need it

**Review by andrea-manzi — APPROVED** [review]
thanks!

**Pull request merged**